python/connect_email.py

```diff
@@ -0,0 +1,64 @@
+#import library 
+import xlrd
+import time
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+"""
+    This file is for send reminder email to users.
+"""
+
+"""
+    have a excel to store dummmy data 
+"""
+path = "clients.xlsx"
```

Comment:
It's great to check from Xlsx to know that it's working or not, but if we can implement to get data from mongo db and send reminders to users from there it will be best.

Reply:
Yes, I did get data from mongo db